src/clib/xtg/surf_get_z_from_xy.c

```diff
@@ -26,18 +26,20 @@
  *    nx, ny        i      Dimensions
  *    xori, yori    i      Map origins
  *    xinc, yinc    i      Map increments
+ *    yflip         i      1 or -1 for map Y axis flip
  *    rot_deg       i      Rotation
  *    p_map_v       i      Pointer to map values to update
- *    flag          i      Flag for options
+ *    option        i      0: interpolation using relative coordinates (rotation is ok!)
+ *                         1: interpolation using rotated map directly
+ *                         2: Nearest node sampling
  *
  * RETURNS:
  *    Z value at point
  *
  * TODO/ISSUES/BUGS:
- *    - checking the handling of undef nodes; shall return UNDEF
- *    - Propert handling of YFLIP = -1!
+
  * LICENCE:
- *    cf. XTGeo LICENSE
+ *    LGPLv3 (cf. XTGeo license)
```

Comment:
Dual licensed??

Reply:
See that it can be misunderstood; will change :+1: